samples/openvino/predict/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenVINO predict pipeline
+
+This is an example of simple one step pipeline implementation including OpenVINO predict component.
```

Comment:
This file is still in the /samples directory. Could we move it to the /contrib directory?

Reply:
samples are now also moved to contrib